.github/workflows/test.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         uses: ruby/setup-ruby@v1
         with: # ruby version specified in .ruby-version
           bundler-cache: true # Automagically runs bundle install
-          cache-version: 1    # Invalidate originally used corrupted cache
```

Comment:
The old bundler cache will not contain the "ancestry" gem, so we need to do this so the new cache is correct.